docs/rules/escape-user-inputs-in-sql-queries.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Escape User Inputs in SQL Queries
+ruleId: EscapeUserInputsInSQLQueries
+since: 3.17.0
+minJavaVersion: 1.1
+remediationCost: 10
+links:
+    - displayName: "Prepared Statements (with Parameterized Queries)"
+      url: "https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html#defense-option-4-escaping-all-user-supplied-input"
+    
+description:
+    Rule description.
+tags: ["Security"]
+---
+
+# Escape User Inputs in SQL Queries
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+This rule prevents Injection by escaping user input before putting it in a SQL-query. It can be applied when an ORACLE-database management system is used. 
```

Comment:
We can mention some more details here: e.g:
 * Which libraries are required,
 * What do we use for escaping the user input (Codec, ESAPI...)